index.html

```diff
@@ -233,18 +238,65 @@ <h1>About Myself</h1>
                   </div>
                 </div>
               </li>
+              <hr />
               <li>
                 <div class="skill-card">
                   <div class="s-title"><h2>Skills</h2></div>
                   <div class="s-icon">
                     <i class="fa-sharp fa-solid fa-angle-right"></i>
                   </div>
                 </div>
+                <hr />
               </li>
             </ul>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Contact Section -->
+
+    <section class="contact">
+      <div class="conatainer grid">
+        <div class="flex">
+          <div class="contact-title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="contact-text">
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hesitate to contact me.
+            </p>
+          </div>
```

Comment:
**1-** Please consider using the same text for the heading as being used in the design which is `Contact me` not `About Myself`.

**2-** Moreover, consider using the same styles for the sub text like the same `font-size, letter-spacing, line-height, width, height, etc` to make it more closer to the design.

## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/228500106-f2a9b063-5fd5-43c3-bd94-5b11ce473362.png)


## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/228500146-94dd282a-4ffc-43a6-8e57-087a8f069f3a.png)